src/core/generators/query.ts

```diff
@@ -417,9 +417,7 @@ ${
   config: any,
   args: infer P,
 ) => any
-  ? P extends unknown
-  ? Record<string, any>
-  : P
+  ? P
```

Comment:
What is the problem with the current type?

Reply:
https://github.com/anymaniax/orval/issues/150